src/App.jsx

```diff
@@ -1,7 +1,7 @@
 import { useState } from "react";
 import styled from "styled-components";
-import { SearchBar } from "./components/SearchBar/";
-import { SearchResults } from "./components/SearchResults/";
+import { SearchBar } from "./components";
+import { SearchResults } from "./components";
```

Comment:
These two lines can be combined into one. That's the whole idea of having a single entry point for importing.

`import { SearchBar, SearchResults } from "./components";`

Reply:
That totally makes sense! Thanks